Set-SslSecurity.ps1

```diff
@@ -138,14 +138,6 @@ process {
     # Connect to Nexus
     Connect-NexusServer -Hostname $SubjectWithoutCn -Credential $Credential -UseSSL
 
-    # Add updated scripts to raw repo in Nexus
-
-    # Push ChocolateyInstall.ps1 to raw repo
-    $ScriptDir = "$env:SystemDrive\choco-setup\files\scripts"
```

Comment:
Got the following error when testing this script

```Get-Content : Cannot find path 'C:\ClientSetup.ps1' because it does not exist.
At C:\choco-setup\files\Set-SslSecurity.ps1:143 char:6
+     (Get-Content -Path $ClientScript) -replace "{{hostname}}", $Subje ...
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\ClientSetup.ps1:String) [Get-Content], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand
```

Looks like this is not getting set within the script.

Reply:
Intrigue. Possibly a rebasing issue from the recently merged branches? I'll look into it in the morning.

Reply:
You were fully right! Not sure how I managed to clean that up. Should be sorted.